hedera-node/src/main/java/com/hedera/services/store/models/TopicConversion.java

```diff
@@ -35,13 +35,29 @@ public static Topic fromMerkle(final MerkleTopic merkleTopic, final Id id) {
 		return modelTopic;
 	}
 
+	/**
+	 * <p>Maps the model topic's fields with the fields of the Merkle topic</p>
+	 * @param model
+	 * 		<p>The source model topic</p>
+	 * @param merkleTopic
+	<p>The targeted Merkle topic</p>
+	 */
+	public static void mapModelChangesToMerkle(final Topic model, final MerkleTopic merkleTopic) {
+		modelToMerkle(model, merkleTopic);
+	}
+
+	//TODO: Fix the failing test.
```

Comment:
Is this TODO an outstanding issue or it was resolved and a leftover?

Reply:
No, still an outstanding issue.